internal/cli/command/cluster/integratedservice/services/securityscan/common.go

```diff
@@ -59,8 +59,10 @@ func (s ServiceSpec) Validate() error {
 
 	validationErrors = errors.Combine(validationErrors, s.WebhookConfig.Validate())
 
-	if s.Registry != nil {
-		validationErrors = errors.Combine(validationErrors, s.Registry.Validate())
+	if s.Registries != nil {
```

Comment:
Note: unnecessary nil check (for loop does it automatically).

Reply:
done